.github/workflows/on-release.yaml

```diff
@@ -0,0 +1,41 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
```

Comment:
Requires making a GH release, which I hadn't done before but was very nice.

Reply:
I'll release v0.1.0 to crates.io and GH as soon as we think this is ready.